CHANGELOG.md

```diff
@@ -8,6 +8,53 @@ Changes since last non-beta release.
 
 *Please add entries here for your pull requests.*
 
+## 9.0 from 8.x. Upgrade Instructions
+All 9.0.0 beta versions can be viewed in [PR 908](https://github.com/shakacode/react_on_rails/pull/908)
+
+- Update the gemfile. Switch over to using Webpacker on the ShakaCode branch:
+
+```rb
+gem "webpacker", git: "https://github.com/shakacode/webpacker.git",
+    branch: "issue-464-merge-webpacker-lite-into-webpacker"
+```
+
+- Update for the renaming in the `WebpackConfigLoader` in your webpack configuration.
+  You will need to rename the following object properties:
+  - hotReloadingUrl       ==> devServerUrl         
+  - hotReloadingHostname  ==> devServerHost        
+  - hotReloadingPort      ==> devServerPort
+
+- Find your `webpacker_lite.yml` and rename it to `webpacker.yml`
```

Comment:
https://github.com/rails/webpacker/blob/master/lib/install/config/webpacker.yml#L8

Default is false.